ethereum/migrations/2_next.js

```diff
@@ -29,6 +29,10 @@ const channels = {
 
 module.exports = function(deployer, network, accounts) {
   deployer.then(async () => {
+
+    // Account of governance contract (to be implemented...)
+    const administrator = accounts[0];
```

Comment:
put in config plz

Reply:
That's actually just stub code until I develop the governance contract and deploy it. No need for config